packages/cli/src/Studio.ts

```diff
@@ -97,20 +96,19 @@ ${bold('Examples')}
 
     loadEnvFile({ schemaPath: args['--schema'], printMessage: true })
 
-    const schemaPath = await getSchemaPathAndPrint(args['--schema'])
+    const { schemaPath, schemas } = await getSchemaPathAndPrint(args['--schema'])
 
     const hostname = args['--hostname']
     const port = args['--port'] || (await getPort({ port: getPort.makeRange(5555, 5600) }))
     const browser = args['--browser'] || process.env.BROWSER
 
     const staticAssetDir = path.resolve(__dirname, '../build/public')
 
-    const schema = await fs.promises.readFile(schemaPath, 'utf-8')
-    const config = await getConfig({ datamodel: schema, ignoreEnvVarErrors: true })
+    const config = await getConfig({ datamodel: schemas, ignoreEnvVarErrors: true })
 
     process.env.PRISMA_DISABLE_WARNINGS = 'true' // disable client warnings
     const studio = new StudioServer({
-      schemaPath,
+      schemaPath: path.resolve(process.cwd(), schemaPath),
```

Comment:
I think we don't need this anymore since `schemaPath` is now always absolute.
```suggestion
      schemaPath
```